src/app/dropdown/dropdown-angular-close-item-false.demo.html

```diff
@@ -5,12 +5,12 @@
   -->
 
 <div class="clr-example squeeze dropdown-demo">
-    <clr-dropdown [clrMenuPosition]="'bottom-right'" [clrCloseMenuOnItemClick]="false">
+    <clr-dropdown [clrCloseMenuOnItemClick]="false">
```

Comment:
I'm not sure about that one. I considered it, but it could make sense on the dropdown, the menu or even the items themselves. I left it on the menu for now, and we could potentially move it, but I'm almost sure we can do that without a breaking change if we need to. So I thought we'd come back to it later.